Sources/HaCWebsiteLib/GitUtils.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+public struct GitUtils {
+  /**
+    Executes a shell command synchronously, returning the exit code
+    
+    - parameters:
+      - in: The path to the directory in which to execute the command
```

Comment:
Good catch, changed this later on. Fun story, the currentDirectoryPath property on Process does absolutely nothing on Linux and that's not documented anywhere 🤷‍♀️